.gitlab-ci.yml

```diff
@@ -17,10 +19,8 @@ test:
   image: golang:1.17
   before_script:
     - apt-get update && apt-get install -yyq liblzma-dev libssl-dev libglib2.0-dev dbus clang-format-9
-    - make get-tools
   script:
     - git ls-tree -r --name-only HEAD | grep -v vendor/ | grep '\.[ch]$' | xargs clang-format-9 -i
-    - make extracheck
```

Comment:
Should we remove the target from the makefile also?

Reply:
We could... but it is harmless to leave and maybe some of us are using it. I can remove it if you insist.

Reply:
No insistence :)